src/js/view/view.js

```diff
@@ -528,7 +528,9 @@ function View(_api, _model) {
         } else {
             // Request media element fullscreen (iOS)
             const provider = model.getVideo();
-            provider.setFullscreen(state);
+            if (provider) {
+                provider.setFullscreen(state);
+            }
```

Comment:
You're supposed to set a provide with instream.

Reply:
Link the FW PR please

Reply:
We don't have fullscreen for IMA for iOS < 12.
For iOS 12 (ipad), the fullscreen works because `fullscreenHelpers.supportsDomFullscreen()` is true.
The `applyProviderListeners` function calls `_setProvider` in `ad-program-controller.js`, but that function doesn't set anything to the provider being passed in: https://github.com/jwplayer/jwplayer/blob/master/src/js/program/ad-program-controller.js#L97
To call `model.setProvider` with the instreamProvider, we need to implement several dummy functions (`getName`, `getPlaybackRate`, etc). Even after that, it causes different problems.